projects/canopy/src/lib/spinner/docs/guide.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/Spinner/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# Spinner
+
+<p class="standfirst">A spinner lets the user know that something is happening. They are normally used on form submissions or loading pages when the data retrieval is typically slow.</p>
+
+![example](docs/spinner/example.png)
+
+## Usage
+
+This component should be used in instances where the duration of a process can't be determined.
+
+Import the component in your application:
+
+```
+@NgModule({
+  ...
+  imports: [ ..., LgSpinnerModule ],
+})
+```
+
+The spinner component should always be used together with the LgSrAlertMessageDirective as screen reader users should be notified when something has finished loading.
+
+## How it works
+
+The spinner animates until the process is completed, when it is replaced by content or the user is navigated to the next part of the journey.
+
+When a spinner is used by itself on a page or card, it must always be accompanied by a supporting message. This message should briefly explain the reason for the wait.
+
+A spinner used inside a component such as a button or input doesn't need an accompanying message.
+
+## Dos and Don'ts
+
+### Do
+
+![do](docs/spinner/do.png)
+
+- Do use spinners to help manage user expectations against technical limitations.
```

Comment:
Could you make the `Do` and `Don't` bold please? Alex must have forgotten:

e.g. `- **Do** use spinners to help manage use`

Reply:
done 👍